data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml

```diff
@@ -65,6 +65,17 @@
             <column name="update_actor" value="admin"/>
         </insert>
 
+        <insert tableName="config" schemaName="${schema.name}">
```

Comment:
Is there a way to reduce the duplication?

Reply:
I don't think so. There is no for loop in liquibase.

Reply:
not sure that's possible (https://stackoverflow.com/questions/25840467/liquibase-common-columns)

Reply:
We can change the format to YAML and then use anchors...

Reply:
interesting.. I think we'd need to get some consensus on this with the wider group. You can drop a question in #corda-platform and see what people think.

Keen to get that in without waiting for this though - we can add a tech debt item to address it, if people agree.

Reply:
I don't know if liquidbase is supposed to be imperative (we can have that debate on the channel).